src/codegen/compile.ml

```diff
@@ -10663,6 +10663,13 @@ and compile_prim_invocation (env : E.t) ae p es at =
     Region.load_word8 env ^^
     TaggedSmallWord.msb_adjust Type.Nat8
 
+  | OtherPrim ("regionLoadInt8"), [e0; e1] ->
```

Comment:
why not in line 10659:
``` ocaml
  OtherPrim (("regionLoadNat8" | "regionLoadInt8") as prim), [e0; e1] ->
```
and in line 10664
``` ocaml
    TaggedSmallWord.msb_adjust Type.(if prim = "regionLoadNat8" then Nat8 else Int8)
```

Reply:
and similarly below ...

Reply:
Sure, that works too, just didn't like the repeated test but maybe that's better than repeated code.

Reply:
Done (also for existing stable memory ops)